src/Datadog.Trace/Tracer.cs

```diff
@@ -292,6 +303,29 @@ private static string GetApplicationName()
             }
         }
 
+        private static DogStatsdService CreateDogStatsdClient(TracerSettings settings)
+        {
+            var frameworkDescription = FrameworkDescription.Create();
+            var tracerVersion = Assembly.GetExecutingAssembly().GetName().Version.ToString();
```

Comment:
`Assembly.GetExecutingAssembly().GetName().Version.ToString();`
Feels like it could be simpler, maybe hidden behind a convenience property or even hard coded.
We could add it to the SynchronizeVersions tool if there isn't a constant we can already rely on.

Reply:
We already have the version hard-coded in so many files that we had to write an entire tool to handle it. I would prefer to hard-code the version in _less_ places, not more.

I like the convenience property idea, so I added this to `TracerConstants.AssemblyVersion` and used it here and in `Api` where we also report the tracer version in the http headers. It's not technically a `const`, so open to suggestions about moving it elsewhere. 748ece7d39342bdc4e164aed3c2f7da50ac33b4f

Reply:
We could rename it to something like tracer metadata or tracer values